rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-configuring.adoc

```diff
@@ -0,0 +1,9 @@
+:_content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-nodes-machinepools-configuring"]
+= Configuring machine pools
+:context: rosa-nodes-machinepools-configuring
+
+toc::[]
+
```

Comment:
Assembly should have an intro: https://redhat-documentation.github.io/modular-docs/#assembly-definition

Reply:
@eohartman Is this exception documented somewhere? I'm not aware of it.

Reply:
I have added a description.